app/routes/home.tsx

```
@@ -68,17 +71,34 @@ export let handle: RouteHandle = {
 					width={loaderData.me.size}
 					height={loaderData.me.size}
-					alt=""
+					alt="Logan McAnsh"
 					className="mx-auto size-60 rounded-full"
 					fetchPriority="high"
 					src={loaderData.me.url}
 					srcSet={loaderData.srcSet}
 				/>
+				<script
+					type="application/ld+json"
+					dangerouslySetInnerHTML={{
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)